platform/platform-configuration-migration/src/test/java/org/codice/ddf/configuration/admin/ConfigurationAdminMigrationTest.java

```diff
@@ -119,11 +108,9 @@
     @Mock
```

Comment:
changed    to static final private String CONFIGURATION_FILE_EXTENSION = ".config";